dev-docs/modules/bidViewableIO.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: page_v2
+page_type: module
+title: Module - Bid Viewable with IntersectionObserver
+description: Triggers BID_VIEWABLE event when a rendered PBJS-Bid is viewable according to an approximation of IAB viewability criteria
+module_code : bidViewabilityIO
+display_name : Bid Viewable with IntersectionObserver
```

Comment:
Can we simplify this down to Bid Viewable?  The IntersectionObserver doesn't seem relevant to publishers -- it's a technology detail, right?

Reply:
I dug into this a bit more tonight, and it appears that display_name is only really used on https://docs.prebid.org/download.html as the label on the checkbox. So this really does need to allow a user on the download page to distinguish between this module and the GPT dependant module.  I think it might make sense to change the display_name for both modules (and perhaps rename these markdown files to actually match the module names (bidViewability, not bidViewable)

I also need to add a link to dev-docs/modules/index.md.

If you, or anyone have thoughts on this please speak up. Otherwise i'll do something arbitrary tomorrow and we can figure it out on the next round of review.

Reply:
I see. How about we change the display name of both modules module to distinguish? Maybe something like "Bid Viewable - GAM" and "Bid Viewable - Ad Server Independent" ?

We're going to need a clear message at the top of both docs to help people choose between them. e.g. something like "If GPT is coded into you page, using that module will be more efficient. Otherwise, as long as you've polyfilled the IntersectionObserver API, this module will allow viewability events."

And since I'm not familiar with this polyfill, it might be helpful to have a link somewhere that explains how to do it?

Reply:
Do these modules otherwise do the same thing -- i.e. just fire the BID_VIEWABLE event? Anything else from a consumer's perspective?

Reply:
The other one has a 'firePixels' configuration where a bid could include 'vurls' that would be requested when a bid became viewable, and a way to write a custom function to convert GPT slot to adUnitCode, as well as firing a BID_VIEWABLE event.